ui/imports/shared/popups/send/controls/GasSelector.qml

```diff
@@ -40,8 +41,23 @@ Item {
         // Normal transaction
         Repeater {
             model: root.bestRoutes
+
             StatusListItem {
                 id: listItem
+
+                property double totalGasAmountL1Eth: {
+                    const l1FeeInGWei = modelData.gasFees.l1GasFee
+                    const l1FeeInEth = globalUtils.wei2Eth(l1FeeInGWei, 9)
```

Comment:
It would be nice to eliminate that `globalUtils` because 1) it's access to the backend, 2) it's direct access to the context property from backend. The same operation can be done using `AmountsArithmetics`.

Reply:
It's also not clear why there is this `9` as argument and what does it mean.